scripts/deploy_kakarot.py

```diff
@@ -2,11 +2,7 @@
 import logging
 from asyncio import run
 
-from scripts.constants import (
-    COMPILED_CONTRACTS,
-    ETH_TOKEN_ADDRESS,
-    EVM_ADDRESS,
-)
+from scripts.constants import COMPILED_CONTRACTS, ETH_TOKEN_ADDRESS, EVM_ADDRESS
```

Comment:
is there a reason why this formatting is only picked up now? just curious

Reply:
i think it's the commit I added, the push hook wasn't working due to us changing the name of the org